.github/workflows/release.yaml

```diff
@@ -0,0 +1,49 @@
+name: Create release on github
+
+on:
+  workflow_run:
+    workflows:
+      - Build binaries
+    types:
+      - "completed"
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
This event means that all other workflows succeeded (i.e. build, test, etc.)?

Reply:
The current behaviour of github actions is that if any of the `workflow_run` workflows defined above (line 4-8) succeed, this action gets triggered. The check here validates that no jobs from all workflows had a failure result.